tests/features/logs.feature

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Logs are related to `invoke`, could you move the tests from here to `invoke.feature` ?